doc/tutorials/tutorial_X8.rst

```diff
@@ -59,19 +59,55 @@ quickstart guide.
 Device details
 --------------
 
-Below, we will be submitting a job to run on the chip ``X8_01``. This is an 8 mode chip,
-with the following restrictions:
+Below, we will be submitting a job to run on the chip ``X8_01``.
+
+``X8_01`` is an 8-mode chip, with the following restrictions:
 
 * The initial states are two-mode squeezed states (:class:`~.S2gate`). We call modes 0 to 3 the
-  *signal modes* and modes 4 to 7 the *idler modes*. Two mode squeezing is between the pairs
+  *signal modes* and modes 4 to 7 the *idler modes*. Two-mode squeezing is between the pairs
   of modes: (0, 4), (1, 5), (2, 6), (3, 7).
 
+At this point, only the parameters :code:`r=1`, :code:`phi=0` are allowed for the two-mode
+squeezing gates between any pair of signal and idler modes. Eventually, a range of
+squeezing amplitudes :code:`r` will be supported.
```

Comment:
Might flow better below the two bullet points, i.e., below line 78.

Reply:
Done!